### PR TITLE
[ch9067] result endpoint for field monitoring

### PR DESCRIPTION
**Pull request by domdinicola**

### CODE REVIEW

**Comment by v-urbanovich** [comment]
Hi!
1) We need to show 'Allocation' and 'Humanitarian Marker' in cp outputs list. So we need API for loading CP Outputs list with 'Allocation' 'Humanitarian Marker' fields
image
2)We need to have connection between partner and intervention. Can you add partner ids array to intervention object or intervention id to partner or something else?
image
3)As we understand this is the links. Can you add links to intervention objet or this is something else?
image
4)Where can I find this information? (Expected Result for intervention)
image
5) And this fields also)? ("CP Indicators" and "Target" fields)
image

Thanks!
PS. You can find invision prototype for this page here - https://projects.invisionapp.com/share/D5JAWD4GTCJ#/screens/326435074

**Comment by domdinicola** [comment]
screen shot 2018-12-17 at 4 56 09 pm

**Comment by domdinicola** [comment]
iraq - /api/reports/results/368/full/

**Comment by domdinicola** [comment]
2) now info are displayed by partner (i didn't get it was a unique table!)

**Comment by domdinicola** [comment]
3) image has been attached

**Comment by domdinicola** [comment]
4/5 I'm following up

**Comment by domdinicola** [comment]
5) 
"result_links" filtered by intervention
  "ll_results"
      "name"

**Comment by domdinicola** [comment]
4. 
use "ram_indicators_name" in "ram_indicators"

**Comment by roman-karpovich** [comment]
@domdinicola please also add targets for the ram indicators.

**Comment by domdinicola** [comment]
@roman-karpovich not clear how to get that..
@ntrncic would be good if we can have a look to it together

**Comment by robertavram** [comment]
@domdinicola do we have humanitarian marker name and code in the api.. we just need to capture it and add those fields to the result model.

**Comment by domdinicola** [comment]
[ch9067]

**Comment by domdinicola** [comment]
@robertavram 
HUMANITARIAN_MARKER_NAME and HUMANITARIAN_MARKER_CODE will also become mandatory for outcome right?

**Comment by robertavram** [comment]
> @robertavram
> HUMANITARIAN_MARKER_NAME and HUMANITARIAN_MARKER_CODE will also become mandatory for **outcome** right?

@domdinicola those are at the **Output** level

File: src/etools/applications/reports/synchronizers.py [hunk 1]
```diff
@@ -236,6 +236,8 @@ class ProgrammeSynchronizer(VisionDataSynchronizer):
         ("OUTCOME_DESCRIPTION", "name"),
         ("OUTCOME_START_DATE", "from_date"),
         ("OUTCOME_END_DATE", "to_date"),
+        ("HUMANITARIAN_MARKER_CODE", "humanitarian_marker_code"),
```
Comment (robertavram) [inline src/etools/applications/reports/synchronizers.py]: please move these to `OUTPUT_MAP`

**Pull request merged**